### DISCUSSION TITLE
Low FPS and Games not starting

### DISCUSSION THREAD

**Discussion by CDUD1214** [UseInterstellar/Interstellar#667] [Q&A] [answered]
Hey my games are running on very low FPS around 5-10 frames per second and many games wont even start. I have put my port to public. However I am on a cheap Chromebook which might explain it however it is a school Chromebook and other students have the same devices but they can run their games smoothly and have no trouble starting up games. Please help.

**Comment by replitidk** [comment]
check your cpu usage and also maybe try clearing cache and cookies and also restart your chrome book and leave it off for about a hour and try again

**Comment by replitidk** [comment]
but just trying your cache and cookies first and tell me what happens @CDUD1214

**Comment by replitidk** [comment]
also it would help if you would say what model of chrome book you have and also it would help if we know the CPU it has

**Comment by replitidk** [comment]
@CDUD1214

**Comment by replitidk** [comment]
@CDUD1214 is it working now?

**Reply by CDUD1214** [reply]
Yea its working now it fixed most of the games but I think that my computer doesn't have enough ram nor power to run some of the games let alone start them up.

**Reply by replitidk** [reply]
ok go to settings for your Chrome book and then go to about chromeOS and click on diagnostics that should bring up your battery health, CPU usage, and also how much RAM you have once you find how much RAM you have and get back to me on it @CDUD1214

**Reply by replitidk** [reply]
the RAM will be listed as memory in diagnostics it is at the very bottom of the diagnostics page

**Reply by replitidk** [reply]
i have 3.70 gigabytes of RAM and that works fine so we can use that as a benchmark to see if its your RAM or your chrome book is broken and you just need to head to IT and say everything is bad on it and that you need a new one or something

**Comment by 0mhx** [comment, marked as answer]
Chromebook moment, chromebooks are very powerful and have the latest nvidia GPUs
Or it's an issue with your device 

(/j)
